polkadot/node/core/approval-voting/src/persisted_entries.rs

```diff
@@ -213,8 +214,17 @@ impl ApprovalEntry {
 				self.tranches.len() - 1
 			},
 		};
-
-		self.tranches[idx].assignments.push((validator_index, tick_now));
+		// We already know already if we seen already an assignment from this validator
```

Comment:
we already used already already ;)
```suggestion
		// We already know if we have seen an assignment from this validator
```